python/ray/rllib/agents/qmix/qmix_policy_graph.py

```diff
@@ -219,11 +228,10 @@ def compute_actions(self,
             random_actions = Categorical(avail).sample().long()
             actions = (pick_random * random_actions +
                        (1 - pick_random) * masked_q_values.max(dim=2)[1])
-            actions = var_to_np(actions)
-            hiddens = var_to_np(hiddens)
+            actions = actions.numpy()
+            hiddens = [s.numpy() for s in hiddens]
 
-        return (TupleActions(list(actions.transpose([1, 0]))),
-                hiddens.transpose([1, 0, 2]), {})
+        return TupleActions(list(actions.transpose([1, 0]))), hiddens, {}
```

Comment:
is the second `[1, 0, 2]` not needed anymore?

Reply:
Yeah (see comment above), hiddens are handled differently now.